tml/org.testeditor.tml.dsl.tests/src/org/testeditor/tml/dsl/validation/TmlParameterUsageValidatorTest.xtend

```diff
@@ -0,0 +1,170 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.tml.dsl.validation
+
+import com.google.inject.Provider
+import javax.inject.Inject
+import org.eclipse.xtext.junit4.util.ParseHelper
+import org.eclipse.xtext.resource.XtextResourceSet
+import org.junit.Before
+import org.junit.Test
+import org.testeditor.aml.AmlModel
+import org.testeditor.aml.dsl.AmlStandaloneSetup
+import org.testeditor.tml.dsl.tests.parser.AbstractParserTest
+import org.eclipse.xtext.junit4.validation.ValidationTestHelper
+
+class TmlParameterUsageValidatorTest extends AbstractParserTest {
```

Comment:
These tests are covered by TclParameterUsageValidatorTest => I'll delete it, since tcl and tml will merge again into one language (propbably)